package.json

```
@@ -30,7 +30,7 @@
     "tty-browserify": "0.0.0",
     "url": "^0.11.0",
     "util": "^0.10.3",
-    "vm-browserify": "0.0.4"
+    "vm-browserify": "1.0.1"
```

This can use ^ now like the other dependencies. I think a pinned version was used previously because the caret in ^0.0.4 has no effect.

```suggestion
    "vm-browserify": "^1.0.1"
```